cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -670,6 +670,34 @@ pub(crate) fn emit(
             }
         }
 
+        Inst::MovImmM { size, simm64, dst } => {
+            let dst = &dst.finalize(state, sink).with_allocs(allocs);
+            let prefix = LegacyPrefixes::None;
+            let default_rex = RexFlags::clear_w();
+            let default_opcode = 0xC7;
+            let bytes = size.to_bytes();
+
+            let (opcode, rex, size) = match *size {
+                OperandSize::Size8 => (0xC6, default_rex, bytes),
```

Comment:
Add a comment here perhaps that we *don't* need `always_emit_if_8bit_needed()` on the REX flags, because the dst is a memory operand, not a possibly 8-bit register?

Reply:
Agreed, done!